robot_sort/robot_sort.py

```diff
@@ -96,17 +96,54 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+        # if can't move right and light is off
+        if self.can_move_right() == False and self.light_is_on() == False:
+            return self
+        
+        self.set_light_on() # turn light on
+        print("Light ON")
+        # while light is on 
+        while self.light_is_on() == True:
+            # pick up item -- swap
+            if self.can_move_right() == True:
+                self.swap_item()
+            self.set_light_off()    # turn light off
+            # while can move right
+            while self.can_move_right() == True:
+                # move right
+                self.move_right()
+                # if item > position -- == 1
+                if self.compare_item() == 1:
+                    # swap
+                    self.swap_item()
```

Comment:
When you do `self.swap_item()` here, the robot will hold the item that you want to swap but not actually swap the two items. You would need some extra steps for the two items that are next to each other to be swapped.